### PR TITLE
fix: add missing labels in standard metrics

### PR DESCRIPTION
**Pull request by Xunzhuo**
Refer to: https://github.com/istio/proxy/blob/master/extensions/common/istio_dimensions.h#L27-L52

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
Waiting on an approval from @istio/wg-user-experience-maintainers as they own the page.

**Review by douglas-reid — CHANGES_REQUESTED** [review]
*Downstream IP* is definitely not a standard label for Istio telemetry.

**Comment by douglas-reid** [comment]
I think this PR is confusion dimensions in stackdriver access logs with dimensions exposed by standard metrics.

File: content/en/docs/reference/config/metrics/index.md [hunk 1]
```diff
@@ -117,4 +117,9 @@ For TCP traffic, Istio generates the following metrics:
...
 
 *   **Source Cluster**: This identifies the cluster of the source workload.
-    This is set by: `global.multiCluster.clusterName` at cluster install time.
\ No newline at end of file
+    This is set by: `global.multiCluster.clusterName` at cluster install time.
+
+*   **gRPC Response Status**: This identifies the response status of the gRPC. This
+    label is present only on gRPC metrics.
+
+*   **Downstream IP**: This identifies the downstream ip.
```
Comment (douglas-reid) [inline content/en/docs/reference/config/metrics/index.md:125]:
```suggestion
```

**Comment by Xunzhuo** [comment]
Thanks @douglas-reid

**Review by douglas-reid — APPROVED** [review]

**Pull request merged**